src/lib/ManticoreBackup.php

```diff
@@ -32,31 +33,60 @@ public static function store(ManticoreClient $Client, FileStorage $Storage, arra
 
     // TODO: add progress bar / backup status reporting
 
-    // If we have no indexes passed we should to query the client and get all indexes we have
-    [$is_all, $indexes] = static::validateIndexes($indexes, $Client);
+    // If we have no tables passed we should to query the client and get all tables we have
+    [$is_all, $tables] = static::validateTables($tables, $Client);
 
-
-    // 0. backup config files
-    echo 'Backing up config files…' . PHP_EOL;
+    // - backup config files
+    println(LogLevel::Info, 'Backing up config files...');
     $is_ok = $Storage->copyPaths([
       $Client->getConfig()->path,
       $Client->getConfig()->schema_path,
     ], $destination['config']);
-    echo '  config files – ' . ($is_ok ? 'OK' : 'FAIL') . PHP_EOL;
+    println(LogLevel::Info, '  config files – ' . get_op_result($is_ok));
+
+    $result = true;
 
-    // 0.5 Lock all indexes to make sure that we will have no new data there
+    // We do backing up state and external files first because it's small enough
+    if ($is_all) {
+      // - Backup global state files
+      println(LogLevel::Info, 'Backing up global state files...');
+      $files = $Client->getConfig()->getStatePaths();
+      $is_ok = $Storage->copyPaths($files, $destination['state'], true);
+      println(LogLevel::Info, '  global state files – ' . get_op_result($is_ok));
+
+      // - backup external files for each index
+      println(LogLevel::Info, 'Backing up external index files...');
+      foreach ($tables as $index => $type) {
+        println(LogLevel::Info, '  ' . $index . ' (' . $type . ')...');
+        // Show settings on distributed tables returns error 500 so we simply skip it
+        if ($type === 'distributed') {
+          println(LogLevel::Info, '   ⏩ SKIP');
```

Comment:
Didn't we want to remove emojis so the output looks nice in `less` etc?

Reply:
It seems like this is the only one emoji that I've missed, will fix it